src/include/kdbnotification.h

```diff
@@ -1,7 +1,37 @@
 /**
  * @file
  *
- * @brief Elektra-Notification structures and declarations
+ * @brief Elektra-Notification structures and declarations for
+ * application developers
+ * 
+ * @ingroup kdbnotification
+ *
+ * @defgroup kdbnotification Notification
+ *
+ * @par Global Mounting
```

Comment:
Thanks for writing the docu.

For uniform style we should put the definition of the group in a comment block after the inclusion of header files, see e.g. in src/libs/elektra/kdb.c.

If there are reasons for your style (e.g. can the `@file` be within a group? Is this useful?) please say so, we might change the other files then.

Reply:
Sure, I changed that.

Yes, a file can be in a group with the `@ingroup` command. Files are then listed in a Doxygen module page before functions - I used this for the I/O and Notification pages.